pysd/vensim2py.py

```diff
@@ -451,7 +454,7 @@ def parse_general_expression(element, namespace=None, subscript_dict=None, macro
     expr_type = array / expr / empty
     expr = _ pre_oper? _ (lookup_def / build_call / macro_call / lookup_call / call / parens / number / reference) _ (in_oper _ expr)?
 
-    lookup_def = ~r"(WITH\ LOOKUP)"I _ "(" _ reference _ "," _ "(" _  ("[" ~r"[^\]]*" "]" _ ",")?  ( "(" _ expr _ "," _ expr _ ")" _ ","? _ )+ _ ")" _ ")"
+    lookup_def = ~r"(WITH\ LOOKUP)"I _ "(" _ expr _ "," _ "(" _  ("[" ~r"[^\]]*" "]" _ ",")?  ( "(" _ expr _ "," _ expr _ ")" _ ","? _ )+ _ ")" _ ")"
```

Comment:
Can you modify the test in the test repo (maybe just add a new variable with a lookup) that exercises this? Its a good find, and I'd love to capture it.